cli/src/lib/npm/npmProjectUtils.js

```diff
@@ -90,6 +97,18 @@ export function getPackageJsonDependencies(
         if (deps[pkgName]) {
           console.warn(`Found ${pkgName} listed twice in package.json!`);
         }
+        const pkgIgnored = ignoreDefs.reduce((acc, cur) => {
+          if (acc) return acc;
+          const ignoreDef = cur.trim();
+          if (ignoreDef === '') return acc;
+          // if we are looking to ignore a scope dir
+          if (ignoreDef.charAt(0) === '@' && ignoreDef.indexOf('/') === -1) {
```

Comment:
Does this mean `@foo/` would not ignore all subfolders? I think this would surprise some people... maybe we could make that more resilient and add a test for it?